### DISCUSSION TITLE
Code block transition between slides

### DISCUSSION THREAD

**Discussion by JoeyGrajciar** [slidevjs/slidev#1045] [Q&A] [answered]
Hi, is there some nice way of doing PowerPoint's Morph with words or Keynote's Magic move transitions over code block?
There is auto animate in revealjs which does this.
Something similar should be also in marp.

**Comment by ragavkumarv** [comment, marked as answer]
Now we have [magic-move](https://sli.dev/guide/syntax#shiki-magic-move) which helps with the above
